apps/onnx/denormal_disabler.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DENORMAL_DISABLER_H_
+#define DENORMAL_DISABLER_H_
+
+#ifdef __SSE__
+#include <xmmintrin.h>
+#endif
+
+class DenormalDisabler {
+public:
+    DenormalDisabler() {
+        // Record how denormals are currently handled so we can restore the current
+        // behavior when we exit the current scope and the class destructor is
+        // called. Note that the current code relies on the CSR register, which is
+        // available on all new x86 cpus.
```

Comment:
Nit: replace 'new' with the actual baseline you expect.

Reply:
Done